packages/core/src/App/Components/Layout/Header/platform-dropdown.jsx

```diff
@@ -28,6 +28,7 @@ const PlatformDropdownContent = ({ platform, app_routing_history }) => {
                 exact={platform.link_to === routes.trade}
                 className='platform-dropdown__list-platform'
                 isActive={() => getActivePlatform(app_routing_history) === platform.name}
+                onClick={e => window.location.pathname.startsWith(platform.link_to) && e.preventDefault()}
```

Comment:
In production, page reload and component remounting are not happening if the clicked platform is already open. So we don't need to set the URL to `pathname` (without `search` params) from `platform_config` in such a case. Using `e.preventDefault()` on `NavLink` allows to achieve this.